doc/string.rdoc

```diff
@@ -48,6 +48,42 @@ class String
     Primitive.rb_str_init(str, encoding, capacity)
   end
 
+  #  call-seq:
+  #    bytes -> array_of_bytes
+  #
+  #  Returns an array of the 8-bit bytes in +self+:
```

Comment:
I feel it's kind of redundant to say `8-bit bytes` since basically all modern systems have `1 byte == 8 bits`. If the system has `1 byte != 8 bits` I think this would return each byte (and not each 8 bits).

Reply:
Done.